tasks/config_server.yml

```diff
@@ -100,3 +112,45 @@
     - rabbitmq_erlang_config | d("") | trim | length == 0
       or rabbitmq_erlang_config == None
     - __rabbitmq_cluster_nodes | d("") | trim | length == 0
+
+- name: "[RabbitMQ] Copy custom server config for ssl internode communication (erlang format)"
+  copy:
+    content: "{{ rabbitmq_internode_ssl_config }}"
+    dest: "/etc/rabbitmq/inter_node_tls.config"
+    force: true
+    mode: 0644
+    owner: root
+  notify:
+    - restart rabbitmq
+  when:
+    - (rabbitmq_internode_ssl_config | d("") | trim | length > 0 and rabbitmq_internode_ssl_config != None)
+      and __rabbitmq_cluster_nodes | d("") | trim | length != 0
+
+- name: Retrieve erl_ssl_path
+  command: erl -noinput -eval 'io:format("ERL_SSL_PATH=~s~n", [filename:dirname(code:which(inet_tls_dist))])' -s init stop
+  register: _erl_ssl_path
+  changed_when: false
+  when:
+    - (rabbitmq_internode_ssl_config | d("") | trim | length > 0 and rabbitmq_internode_ssl_config != None)
+      and __rabbitmq_cluster_nodes | d("") | trim | length != 0
+
+- name: Write rabbitmq env conf
+  template:
+    src: "{{ rabbitmq_env_tpl }}"
+    dest: "/etc/rabbitmq/rabbitmq-env.conf"
+    force: true
+    mode: 0644
+    owner: root
+  when:
```

Comment:
this step is not playing for me.
I have set rabbitmq_internode_ssl_config.

```
TASK [rockandska.rabbitmq : [RabbitMQ] Copy custom server config for ssl internode communication (erlang format)] **********************
changed: [p-bifrost-amqp-002]
changed: [p-bifrost-amqp-003]

TASK [rockandska.rabbitmq : Retrieve erl_ssl_path] *************************************************************************************
ok: [p-bifrost-amqp-002]
ok: [p-bifrost-amqp-003]

TASK [rockandska.rabbitmq : Write rabbitmq env conf] ***********************************************************************************
skipping: [p-bifrost-amqp-002]
skipping: [p-bifrost-amqp-003]
```

Reply:
Which ansible version do you use ?
Because I don't change `rabbitmq_env_config` in molecule scenario and everything is fine with 2.9, 2.10

Reply:
this resolve the skipping of the task but the file is empty because it use the same test.

```
{% if _erl_ssl_path.skipped is false %}
```

Reply:
In fact you're right
https://github.com/rockandska/ansible-role-rabbitmq/runs/4178571951?check_suite_focus=true#step:3:881

I missed this one....
So the clustering is running without SSL.....
Not sure if in `cluster_status` there is something who show that the internode communication is SSL or not to add a test on it

Reply:
for me clustering didn't start..

Reply:
Join me on [Gitter](https://gitter.im/rockandska) if you want, but the cluster scenario works well with your modification.

Reply:
As discussed on Gitter:
- the task was updated with your suggestion and works fine
- I update the molecule cluster scenario to [check that the internode communication is done with SSL](https://github.com/rockandska/ansible-role-rabbitmq/pull/25/files#diff-5af8b87588727ccdfcf4c6715fcdce7748b11e40800038950b714143c73dfbcfR11-R25)